config/installation_config.yml

```diff
@@ -41,67 +56,138 @@
   display_title: 'Chatwoot Metadata'
   description: 'Display default Chatwoot metadata like favicons and upgrade warnings'
   type: boolean
-- name: MAILER_INBOUND_EMAIL_DOMAIN
-  value:
-  locked: false
-- name: MAILER_SUPPORT_EMAIL
-  value:
+# ------- End of Branding Related Config ------- #
+
+
+
+# ------- Signup & Account Related Config ------- #
+- name: ENABLE_ACCOUNT_SIGNUP
+  display_title: 'Enable Account Signup'
+  type: boolean
+  description: 'Allow users to signup for new accounts'
+  value: false
+  description: 'Allow users to signup for new accounts'
   locked: false
 - name: CREATE_NEW_ACCOUNT_FROM_DASHBOARD
   value: false
+  description: 'Allow users to create new accounts from the dashboard'
   locked: false
-- name: INSTALLATION_EVENTS_WEBHOOK_URL
+- name: HCAPTCHA_SITE_KEY
   value:
   locked: false
-- name: CHATWOOT_INBOX_TOKEN
+- name: HCAPTCHA_SERVER_KEY
   value:
   locked: false
-- name: CHATWOOT_INBOX_HMAC_KEY
+- name: INSTALLATION_EVENTS_WEBHOOK_URL
   value:
+  display_title: 'System events Webhook URL'
+  description: 'The URL to which the system events like new accounts created will be sent'
```

Comment:
Same comments as above. Let us not expose this to the end user. This is only used internally.

Reply:
Added description here so that we have a cue of what this config is used for. removed showing this from the UI